TESTS/netsocket/README.md

```diff
@@ -1654,6 +1669,311 @@ through a dedicated socket
 Echo server returns data to both threads and received data matches to
 send data. The additional thread isn't stopped prematurely.
 
+Test cases for DNS class
+---------------------------
+
+### ASYNCHRONOUS_DNS
+
+**Description:**
+
+Verify basic functionality of asynchronous DNS. Call `NetworkInterface::gethostbyname_async()` with a valid host name and verify result.
+
+**Preconditions:**
+
+1.  Network interface is initialised.
+2.  Network connection is up.
+
+**Test steps:**
+
+1.  Call `gethostbyname_async()` with a valid host name and a callback.
+2.  Verify that callback is called with correct parameters.
+
+**Expected result:**
+
+Callback is called with NSAPI_ERROR_OK and IP address.
+
+### ASYNCHRONOUS_DNS_SIMULTANEOUS
+
+**Description:**
+
+Verify that simultaneous asynchronous DNS queries work correctly. Call `NetworkInterface::gethostbyname_async()` in a row 6 times with a different host names. Wait for all requests to complete and verify result. Cache shall not contain host names used in asynchronous request.
```

Comment:
"...to complete, and verify..."

Reply:
I don't understand what the last sentence means. It shouldn't contain those? It can't contain those? Please clarify.